### DISCUSSION TITLE
Image not showing up on github pages.

### DISCUSSION THREAD

**Discussion by da-Jester** [Chiroyce1/Chiroyce1#15] [Q&A] [answered]
Hello! So I created a [this repl](https://replit.com/@-Krios-/Website-Beta?v=1) as a temporary cover page for my website while I build a wordpress site. I want to host it on github pages. So using replit I created a [github repo for the project](https://github.com/da-Jester/Jesterhat.org-Placeholder/) and set up github pages. but for some reason, the image doesn't show up on the github pages site. it's fine on replit, and I can see the image is on github (just go to /files/jester-hello), but when I open the website, it doesn't show up. I was wondering if you have any ideas about why this is happening.

**Comment by Chiroyce1** [comment, marked as answer]
Ah, thats because of the baseurl issue, on github pages your site has a prefix of `/Jesterhat.org-Placeholder/` whereas the repl doesn't use that prefix. A simple fix is to make the image urls point to the exact path using the `./` prefix. For example, the image on the homepage is linked via `files/jester-hello.png`, which wouldn't work, so in order for it to be converted into `/Jesterhat.org-Placeholder/files/jester-hello.png`, just link it via `./files/jester-hello.png`.

Hope this works, otherwise I can take another look at it (I just assumed this was the issue as its very common)

**Reply by da-Jester** [reply]
thanks, I'll try it

**Reply by da-Jester** [reply]
it worked!
